src/theme/colors.ts

```diff
@@ -1,6 +1,8 @@
 export const palette = {
   coral: '#f73a52',
   darkBlue: '#05001e',
+  darkBlueAlt: `#0f0d34`,
+  midBlue: `#18174a`,
```

Comment:
🔧 Backticks are unnecessary here.
```suggestion
  darkBlueAlt: '#0f0d34',
  midBlue: '#18174a',
```

Reply:
This must be a linter thing, all of the colors in my local have backticks and I'm getting an error when I remove them.